Documentation/AdvancedBuildInstructions.md

```diff
@@ -100,6 +100,38 @@ By default, ccache will include the plugins themselves in file hashes. So if a p
 will change, and you will be stuck with an uncached build. This setting will prevent ccache from using plugins in the
 file hashes.
 
+## Building the flatpak
+
+Ladybird has an in-tree flatpak manifest that can be used to build a flatpak package for Ladybird. The recommended way
+to build the flatpak is to use the `flatpak-builder` tool, which is available in most distributions. See the
+[flatpak setup documentation](https://flatpak.org/setup/) on how to configure your environment for user flatpak
+builds, and to configure the flathub repo. The Ladybird flatpak manifest at
```

Comment:
Should Flathub be used instead? To bring it in line with how Ladybird and CMake are capitalized according to the project naming.

Reply:
Yeup, Flathub's guidelines say capital F for this as well.